docs/components/DocumentationPage.tsx

```diff
@@ -263,87 +264,93 @@ export default class DocumentationPage extends React.Component<Props, State> {
     const algoliaTag = this.getAlgoliaTag();
 
     return (
-      <DocumentationNestedScrollLayout
-        ref={this.layoutRef}
-        header={headerElement}
-        sidebar={sidebarElement}
-        sidebarRight={sidebarRight}
-        tocVisible={this.props.tocVisible}
-        isMenuActive={this.state.isMenuActive}
-        isMobileSearchActive={this.state.isMobileSearchActive}
-        onContentScroll={handleContentScroll}
-        sidebarScrollPosition={sidebarScrollPosition}>
-        <Head title={`${this.props.title} - Expo Documentation`}>
-          {algoliaTag !== null && <meta name="docsearch:version" content={algoliaTag} />}
-          <meta property="og:title" content={`${this.props.title} - Expo Documentation`} />
-          <meta property="og:type" content="website" />
-          <meta property="og:image" content="https://docs.expo.io/static/images/og.png" />
-          <meta property="og:image:url" content="https://docs.expo.io/static/images/og.png" />
-          <meta
-            property="og:image:secure_url"
-            content="https://docs.expo.io/static/images/og.png"
-          />
-          <meta property="og:locale" content="en_US" />
-          <meta property="og:site_name" content="Expo Documentation" />
-          <meta
-            property="og:description"
-            content="Expo is an open-source platform for making universal native apps for Android, iOS, and the web with JavaScript and React."
-          />
-
-          <meta name="twitter:site" content="@expo" />
-          <meta name="twitter:card" content="summary" />
-          <meta property="twitter:title" content={`${this.props.title} - Expo Documentation`} />
-          <meta
-            name="twitter:description"
-            content="Expo is an open-source platform for making universal native apps for Android, iOS, and the web with JavaScript and React."
-          />
-          <meta property="twitter:image" content="https://docs.expo.io/static/images/twitter.png" />
-
-          {(version === 'unversioned' || this.isPreviewPath()) && (
-            <meta name="robots" content="noindex" />
-          )}
-          {version !== 'unversioned' && <link rel="canonical" href={this.getCanonicalUrl()} />}
-        </Head>
-
-        {!this.state.isMenuActive ? (
-          <div css={STYLES_DOCUMENT}>
-            <H1>{this.props.title}</H1>
-            <DocumentationPageContext.Provider value={{ version }}>
-              {this.props.children}
-            </DocumentationPageContext.Provider>
-            <DocumentationFooter
-              title={this.props.title}
-              url={this.props.url}
-              asPath={this.props.asPath}
-              sourceCodeUrl={this.props.sourceCodeUrl}
+      <>
+        <SkipToContent>Skip to main content</SkipToContent>
```

Comment:
ℹ️ The whole modification is mostly about adding wrapping the rendered tree inside a fragment so that I can add the `<SkipToContent>` component. The idea is to allow users to navigate directly to the main content without having to tab through all the links of the page.

Before:

![Navigating using tab keys until reaching the main content (no skip to content)](https://user-images.githubusercontent.com/3874873/119313059-62378a00-bc73-11eb-98fc-44af602057c9.gif)


After

![Navigating using tabs element until reaching the main content (using skip to content)](https://user-images.githubusercontent.com/3874873/119313112-74192d00-bc73-11eb-9b38-48fe97e47902.gif)
---

The main idea is to allow people navigating without a mouse to directly reach the main content of the website without having to browse every single link before the <main> tag.